.github/CODEOWNERS

```diff
@@ -1,12 +1,9 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/global.json                            @aspnet/build
-/azure-pipelines.yml                    @aspnet/build
-/eng/                                   @aspnet/build
-/eng/common/                            @dotnet-maestro-bot @dougbu
-/eng/Versions.props                     @dotnet-maestro-bot @dougbu
-/eng/Version.Details.xml                @dotnet-maestro-bot @dougbu
+/eng/common/                            @dotnet-maestro-bot
+/eng/Versions.props                     @dotnet-maestro-bot
+/eng/Version.Details.xml                @dotnet-maestro-bot
 /src/Caching/                           @tratcher
 /src/Configuration/                     @tratcher
 /src/Hosting/                           @tratcher
```

Comment:
@Tratcher do you want to keep watching these folders❔

Reply:
Yeah, I'll keep watching.